client/src/components/SearchBar.jsx

```diff
@@ -1,12 +1,72 @@
-import React from "react";
+import React, { useState } from "react";
+import { Grid, IconButton, useTheme } from "@mui/material";
+import SuitcaseIcon from "@mui/icons-material/BusinessCenterOutlined";
+import LocationIcon from "@mui/icons-material/FmdGoodOutlined";
+import SearchIcon from "@mui/icons-material/SearchOutlined";
 
+import IconTextField from "./IconTextField";
 /**
  * https://github.com/BobaTalks/internship-tracker/issues/23
  * Search bar component should include the "what" and "where" inputs with a button to submit
  * When completed, this should be imported into appropriate pages.
  */
 const SearchBar = () => {
-  return <></>;
+  const theme = useTheme();
+  const [internship, setInternship] = useState("");
+  const [location, setLocation] = useState("");
+
+  const handleClick = () => {};
+  return (
+    <Grid
+      container
+      item
```

Comment:
Do you need the item attribute if this is not a child of a container grid?

Reply:
see comment below